include/matx/core/tensor_impl.h

```diff
@@ -57,6 +58,7 @@ struct DenseTensorData {
 
 template <typename T, typename CRD, typename POS, typename TF>
 struct SparseTensorData {
+  using value_type = T;
```

Comment:
shall we move this to L62?

(so we have sparse_data first, then the template params T/CRD/POS/TF)

Reply:
sure